### ISSUE TITLE
Being able to search genres

### ISSUE THREAD

**Issue by ardx812** [recloudstream/cloudstream#400]
### Describe your suggested feature

This would be a great feature because most streaming platforms have this.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Gamer750** [comment]
It would be good . It would make easy to find anime

**Comment by Gamer750** [comment]
Any update

**Comment by dondeepak** [comment]
Yes it will be grate to have search option with genre. Please implement this feature

**Comment by 7ymekk** [comment]
it would be great to filter by all metadata: genre, year, score, type or even tag. I opt for renaming the Issue so it'll support all of the filtering options.
